recipes/ckanbatch-efs-setup.rb

```diff
@@ -0,0 +1,56 @@
+# Sets up EFS and EBS directories and links for CKAN batch layer.
+#
+# Copyright 2020, Queensland Government
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Update EFS Data directory for CKAN
```

Comment:
Could you add more comments, i.e

This sets efs as default, if an extra ebs (non root) is found, it will off load /var/log/ckan and ensure it has correct permissions